### ISSUE TITLE
deprecation warning for "rawsource"

### ISSUE THREAD

**Issue by drammock** [mcmtroffaes/pybtex-docutils#21]
I hit this DeprecationWarning today while one of our CIs was building docs for MNE-Python:

```
Traceback (most recent call last):
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinx/cmd/build.py", line 272, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinx/application.py", line 223, in __init__
    self.setup_extension(extension)
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinx/application.py", line 379, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinx/registry.py", line 430, in load_extension
    mod = import_module(extname)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinxcontrib/bibtex/__init__.py", line 9, in <module>
    from .domain import BibtexDomain
  File "/home/circleci/python_env/lib/python3.8/site-packages/sphinxcontrib/bibtex/domain.py", line 20, in <module>
    import pybtex_docutils
  File "/home/circleci/python_env/lib/python3.8/site-packages/pybtex_docutils/__init__.py", line 35, in <module>
    class Backend(BaseBackend):
  File "/home/circleci/python_env/lib/python3.8/site-packages/pybtex_docutils/__init__.py", line 39, in Backend
    'ndash': [docutils.nodes.Text('\u2013', '\u2013')],
  File "/home/circleci/python_env/lib/python3.8/site-packages/docutils/nodes.py", line 411, in __init__
    warnings.warn('nodes.Text: initialization argument "rawsource" '
DeprecationWarning: nodes.Text: initialization argument "rawsource" is ignored and will be removed in Docutils 1.3.

Exception occurred:
  File "/home/circleci/python_env/lib/python3.8/site-packages/docutils/nodes.py", line 411, in __init__
    warnings.warn('nodes.Text: initialization argument "rawsource" '
DeprecationWarning: nodes.Text: initialization argument "rawsource" is ignored and will be removed in Docutils 1.3.
```

The CI is running the following versions:
<details>

```
python -m pip list

Package                       Version      Editable project location
----------------------------- ------------ -------------------------
aiohttp                       3.8.1
aiosignal                     1.2.0
alabaster                     0.7.12
appdirs                       1.4.4
argon2-cffi                   21.3.0
argon2-cffi-bindings          21.2.0
asttokens                     2.0.5
async-timeout                 4.0.2
attrs                         21.4.0
Babel                         2.10.1
backcall                      0.2.0
beautifulsoup4                4.11.1
bleach                        5.0.0
build                         0.8.0
certifi                       2022.5.18.1
cffi                          1.15.0
cftime                        1.6.0
charset-normalizer            2.0.12
check-manifest                0.48
codespell                     2.1.0
colorspacious                 1.1.2
commonmark                    0.9.1
coverage                      6.4
cryptography                  37.0.2
cycler                        0.11.0
darkdetect                    0.6.0
debugpy                       1.6.0
decopatch                     1.4.10
decorator                     5.1.1
defusedxml                    0.7.1
Deprecated                    1.2.13
dipy                          1.5.0
docutils                      0.18.1
entrypoints                   0.4
executing                     0.8.3
fastjsonschema                2.15.3
flake8                        4.0.1
flake8-array-spacing          0.2.0
fonttools                     4.33.3
frozenlist                    1.3.0
future                        0.18.2
h5io                          0.1.7
h5py                          3.7.0
idna                          3.3
imageio                       2.19.2
imageio-ffmpeg                0.4.7
imagesize                     1.3.0
importlib-metadata            4.11.4
importlib-resources           5.7.1
iniconfig                     1.1.1
ipycanvas                     0.12.0
ipyevents                     2.0.1
ipykernel                     6.13.0
ipython                       8.3.0
ipython-genutils              0.2.0
ipyvtklink                    0.2.2
ipywidgets                    7.7.0
jedi                          0.18.1
jeepney                       0.8.0
Jinja2                        3.1.2
joblib                        1.1.0
jsonschema                    4.5.1
jupyter                       1.0.0
jupyter-client                7.3.1
jupyter-console               6.4.3
jupyter-core                  4.10.0
jupyterlab-pygments           0.2.2
jupyterlab-widgets            1.1.0
keyring                       23.5.1
kiwisolver                    1.4.2
latexcodec                    2.0.1
llvmlite                      0.38.1
lxml                          4.8.0
makefun                       1.13.1
MarkupSafe                    2.1.1
matplotlib                    3.5.2
matplotlib-inline             0.1.3
mccabe                        0.6.1
memory-profiler               0.60.0
mffpy                         0.7.2
mistune                       0.8.4
mne                           1.1.dev0     /home/circleci/project
mne-bids                      0.11.dev0
mne-connectivity              0.4.dev0
mne-qt-browser                0.4.dev0
multidict                     6.0.2
nbclient                      0.6.3
nbconvert                     6.5.0
nbformat                      5.4.0
neo                           0.10.2
nest-asyncio                  1.5.5
netCDF4                       1.5.8
nibabel                       3.2.2
nilearn                       0.9.1
notebook                      6.4.11
numba                         0.55.1
numexpr                       2.8.1
numpy                         1.21.6
numpydoc                      -1.3.2.dev0-
packaging                     21.3
pandas                        1.4.2
pandocfilters                 1.5.0
parso                         0.8.3
patsy                         0.5.2
pep517                        0.12.0
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        9.1.1
pip                           22.1.1
pkg_resources                 0.0.0
pkginfo                       1.8.2
pluggy                        1.0.0
ply                           3.11
pooch                         1.6.0
prometheus-client             0.14.1
prompt-toolkit                3.0.29
psutil                        5.9.1
ptyprocess                    0.7.0
pure-eval                     0.2.2
py                            1.11.0
pybtex                        0.24.0
pybtex-docutils               1.0.1
pycodestyle                   2.8.0
pycparser                     2.21
pydata-sphinx-theme           0.8.1
pydocstyle                    6.1.1
pyflakes                      2.4.0
Pygments                      2.12.0
pymatreader                   0.0.30
pyparsing                     3.0.9
pyqtgraph                     0.12.4
pyrsistent                    0.18.1
PySide6                       6.2.4
pytest                        7.1.2
pytest-cov                    3.0.0
pytest-harvest                1.10.3
pytest-timeout                2.1.0
python-dateutil               2.8.2
python-picard                 0.7
pytz                          2022.1
pyvista                       0.35.dev0
pyvistaqt                     0.10.dev0
pyxdf                         1.16.3
PyYAML                        6.0
pyzmq                         23.0.0
QDarkStyle                    3.0.3
qtconsole                     5.3.0
QtPy                          2.1.0
quantities                    0.13.0
readme-renderer               35.0
requests                      2.27.1
requests-toolbelt             0.9.1
rfc3986                       2.0.0
rich                          12.4.4
scikit-learn                  1.1.1
scipy                         1.8.1
scooby                        0.5.12
seaborn                       0.11.1
SecretStorage                 3.3.2
Send2Trash                    1.8.0
setuptools                    62.3.2
shiboken6                     6.2.4
sip                           6.6.1
six                           1.16.0
snowballstemmer               2.2.0
soupsieve                     2.3.2.post1
Sphinx                        5.0.0b1
sphinx-copybutton             0.5.0
sphinx-gallery                0.11.0.dev0
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-bibtex          2.4.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
stack-data                    0.2.0
statsmodels                   0.13.2
terminado                     0.15.0
threadpoolctl                 3.1.0
tinycss2                      1.1.1
toml                          0.10.2
tomli                         2.0.1
tornado                       6.1
tqdm                          4.64.0
traitlets                     5.2.1.post0
twine                         4.0.0
typing_extensions             4.2.0
urllib3                       1.26.9
vtk                           9.1.0
wcwidth                       0.2.5
webencodings                  0.5.1
wheel                         0.37.1
widgetsnbextension            3.6.0
wrapt                         1.14.1
wslink                        1.6.4
xarray                        2022.3.0
xlrd                          2.0.1
xmltodict                     0.13.0
yarl                          1.7.2
zipp                          3.8.0
```

</details>

Prior to running the CI I was able to build our docs locally without warnings, and a quick spot-check shows that the likely difference is that locally I have docutils 0.17.1 and the CI has docutils 0.18.1.  Local/CI versions of sphinxcontrib-bibtex, pybtex, and pybtex_docutils all match.

**Comment by mcmtroffaes** [comment]
This should be fixed now on develop. Can you check if it works for you?

**Comment by mcmtroffaes** [comment]
Fixed in 1.0.2 (just released on pypi). Thanks again for reporting!

**Comment by drammock** [comment]
thanks for the quick fix!  Our CI fails differently now (also related to docutils 0.18, but not to any of your packages). 👍🏻
